libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -232,6 +236,9 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
               voltage -= 65534.0f;
           }
         }
+        for (uint8_t i = 0; i < MAVLINK_MSG_BATTERY_STATUS_FIELD_VOLTAGES_EXT_LEN; i++) {
```

Comment:
Could use `ARRAY_SIZE()` instead - but this matches the other bit, so meh.

This could also have been accomplished by initialising the array up the top with `{}`.  But again - not like the other bit, so meh.

Reply:
Technically not initializing it above saves runtime, but it's a tiny win... Arguably this part should just be a `memset`. Not critical however...